gitbook-docs/security.md

```diff
@@ -112,6 +124,93 @@ Clients using other kinds of protocols can include the `token` in the Signal K m
 }
 ```
 
+### Token Validation
+
+Tokens may have a short expiry time and need to be renewed periodically, or a token's current validity may be unknown. 
+
+#### HTTP Clients
+
+To validate a token, a web based client should send an HTTP GET request to `/signalk/«version»/auth/validate` with the token in the cookie, or in the header.
```

Comment:
Rereading this one more time...shouldn't this be a POST?

Reply:
There is no body since the data is in the header hence a I used GET. Is POST the correct way?

Reply:
IMHO security related requests should not be GETs so that they are less susceptible to XSRF.